contracts/source/BlockVerifier.sol

```diff
@@ -1,4 +1,4 @@
-pragma solidity 0.6.8;
+pragma solidity >=0.8.0;
```

Comment:
Explicit versions should be used to prevent unexpected/untested problems occurring.  This does cause some difficulty when using libraries, but I think in this case (since money is almost always involved) safety trumps usability.

Reply:
Thoughts on tilde for this @MicahZoltu? Currently assessing this one for our use-case and I can imagine `~0.8` would likely keep it bounded enough to avoid most of the risk while not forcing a hard version fix.